### PR TITLE
Updating nodes/workflows functional tests

### PR DESCRIPTION
**Pull request by BillyAbildgaard**
Functional Test updates need for the https://github.com/RackHD/on-http/pull/313 to pass testing

I tested these changes locally and it fixed all failures seen in the above PR.

@RackHD/corecommitters @dalebremner @srinia6

### CODE REVIEW

**Comment by BillyAbildgaard** [comment]
This PR is dependent on libraries being updated via RackHD/on-http#313

**Comment by srinia6** [comment]
+1

**Comment by brianparry** [comment]
test this please

File: test/tests/api/v2_0/nodes_tests.py [hunk 1]
```diff
@@ -80,24 +80,22 @@ def __init__(self):
...
+	Api().nodes_get_workflow_by_id(identifier=id, active=query )
+	data = self.__get_data()
+	if len(data) > 0:
+	    status = data[0].get('_status')
+	    return status
+	return 'not running'
         
     def __post_workflow(self, id, graph_name):
-        status = self.__get_workflow_status(id)
+        status = self.__get_workflow_status(id, 'true')
```
Comment (yyscamper) [inline test/tests/api/v2_0/nodes_tests.py]: why the query is string not a boolean?

Comment (BillyAbildgaard) [inline test/tests/api/v2_0/nodes_tests.py]: @yyscamper when I tried passing it in as a Boolean it threw errors.  I believe the Boolean is lowercase in python and capitalized in javascript which was messing things up

**Comment by brianparry** [comment]
test this please

**Pull request merged**